src/permission-set.js

```diff
@@ -719,11 +720,12 @@ class PermissionSet {
    * @method save
    * @param [aclUrl] {String} Optional URL to save the .ACL resource to. Defaults
    *   to its pre-set `aclUrl`, if not explicitly passed in.
+   * @param [contentType] {string} Optional content type to serialize it as
    * @throws {Error} Rejects with an error if it doesn't know where to save, or
    *   with any XHR errors that crop up.
    * @return {Promise<Request>}
    */
-  save (aclUrl, rdf, webClient) {
+  save (aclUrl, contentType, rdf, webClient) {
```

Comment:
actually if one can set all above except contentType in the constructor, what use cases you have for overwriting it in save?